schedule/yam/test_cases/installer_extended.yaml

```diff
@@ -7,13 +7,11 @@ description: >
      - License translations (except SLE 15 due to missing translations in the
        beta phase).
      - Check available and selected by default modules for installation.
-vars:
-  CHECK_PRESELECTED_MODULES: '1'
-  CHECK_RELEASENOTES: '1'
-  EXIT_AFTER_START_INSTALL: '1'
-  EXTRABOOTPARAMS: Y2STRICTTEXTDOMAIN=1
-  YUI_REST_API: 1
 schedule:
+  access_beta:
+    - installation/validate_beta_message
+    - installation/access_beta_distribution
+    - installation/language_keyboard/switch_keyboard_layout
```

Comment:
one small detail, this belongs to product selection not to beta.

Reply:
> one small detail, this belongs to product selection not to beta.

That's the tricky part, it we set